include/rocksdb/listener.h

```diff
@@ -251,11 +251,39 @@ class EventListener {
   virtual ~EventListener() {}
 };
 
+struct MemTableInfo {
+  // the name of the column family to which memtable belongs
+  std::string cf_name;
+  // First sequence number in memtable
+  SequenceNumber smallest_seqno;
+};
+
+// Callbacks from this class are called with db mutext held.
```

Comment:
Actually realized that SwitchMemtable happens on the write thread, so I can move the callback to be outside the lock (and still record our internal state, namely replication seq number, correctly). I will update the PR.